quandl/utils/api_key_util.py

```diff
@@ -7,3 +9,9 @@ def init_api_key_from_args(params):
         # set api key
         if 'api_key' in params:
             ApiConfig.api_key = params.pop('api_key')
+        # issue warning if api was previously set with params
+        # and now is not
+        elif (ApiConfig.api_key is not None) and os.isatty(0):
```

Comment:
it was only trigger once, but does not know if the ApiConfig.api_key was set/unset either through the `Quandl.get` method or through another means